docs/user/dql/functions.rst

```diff
@@ -234,9 +234,67 @@ CEIL
 Description
 >>>>>>>>>>>
 
-Specifications:
+Usage: CEIL(T) takes the ceiling of value T.
+
+Note: CEIL and `CEILING`_ functions have the same implementation & functionality
+
+Limitation: CEIL only works as expected when IEEE 754 double type displays decimal when stored.
+
+Argument type: INTEGER/LONG/FLOAT/DOUBLE
+
+Return type: LONG
+
+Example::
+
+   os> SELECT CEIL(0), CEIL(50.00005), CEIL(-50.00005);
+    fetched rows / total rows = 1/1
+    +-----------+------------------+-------------------+
+    | CEIL(0)   | CEIL(50.00005)   | CEIL(-50.00005)   |
+    |-----------+------------------+-------------------|
+    | 0         | 51               | -50               |
+    +-----------+------------------+-------------------+
+
+   os> SELECT CEIL(3147483647.12345), CEIL(113147483647.12345), CEIL(3147483647.00001);
+    fetched rows / total rows = 1/1
+    +--------------------------+----------------------------+--------------------------+
+    | CEIL(3147483647.12345)   | CEIL(113147483647.12345)   | CEIL(3147483647.00001)   |
+    |--------------------------+----------------------------+--------------------------|
+    | 3147483648               | 113147483648               | 3147483648               |
+    +--------------------------+----------------------------+--------------------------+
+
+CEILING
```

Comment:
if the function of ceil and ceiling is same, we can do similar thing like mysql doc https://dev.mysql.com/doc/refman/5.6/en/mathematical-functions.html#function_ceiling

Reply:
Rebased and updated in b4b092b39.